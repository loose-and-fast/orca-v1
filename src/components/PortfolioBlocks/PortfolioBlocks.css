/*.PortfolioBlocks-logo {
	height: 100%;
	width: 95%;
	background-color: black;
	margin: 10px 0px;
}*/

.PortfolioBlocks-water img {
	height: 100%;
	width: 95%;
	background-color: white;
	margin: 10px 0px;
}

.PortfolioBlocks-tree img {
	text-align: center;
	height: 100%;
	width: 90%;
	margin: 10px 0px;
}

.PortfolioBlocks-lion img {
	text-align: center;
	height: 100%;
	width: 90%;
	margin: 10px 0px 0px;
}

.PortfolioBlocks-dice img {
	text-align: center;
	height: 100%;
	width: 90%;
	margin: 10px 0px;
}

.PortfolioBlocks-rose img {
	text-align: center;
	height: 100%;
	width: 90%;
	margin: 10px 0px;
}

.PortfolioBlocks-tower img {
	text-align: center;
	height: 100%;
	width: 90%;
	margin: 10px 0px 0px;
}

.PortfolioBlocks-camera img {
	text-align: center;
	height: 100%;
	width: 90%;
	margin: 10px 0px;
}

.PortfolioBlocks-tracks img {
	text-align: center;
	height: 100%;
	width: 90%;
	margin: 10px 0px;
}

.PortfolioBlocks-clock img {
	text-align: center;
	height: 100%;
	width: 90%;
	margin: 10px 0px 0px;
}