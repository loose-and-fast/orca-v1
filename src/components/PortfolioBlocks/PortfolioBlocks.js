import React from "react";
import PropTypes from "prop-types";
import "./PortfolioBlocks.css";


const PortfolioBlocks = ({ className, imageSrc }) => {
	return (	
	<div className={`PortfolioBlocks-${className}`}>
		<img src={imageSrc} height="100" width="100"/>
	</div>);
}

PortfolioBlocks.propTypes = {};

export default PortfolioBlocks;
