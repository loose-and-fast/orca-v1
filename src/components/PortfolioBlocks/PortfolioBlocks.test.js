import React from "react";
import PortfolioBlocks from "./PortfolioBlocks";

test("PortfolioBlocks renders", () => {
  const props = {};
  const wrapper = shallow(<PortfolioBlocks {...props} />);
  expect(wrapper).toMatchSnapshot();
});
