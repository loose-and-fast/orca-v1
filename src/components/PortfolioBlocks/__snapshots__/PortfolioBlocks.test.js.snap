// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PortfolioBlocks renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PortfolioBlocks />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <img
        height="100"
        src={undefined}
        width="100"
/>,
      "className": "PortfolioBlocks-undefined",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "height": "100",
        "src": undefined,
        "width": "100",
      },
      "ref": null,
      "rendered": null,
      "type": "img",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <img
          height="100"
          src={undefined}
          width="100"
/>,
        "className": "PortfolioBlocks-undefined",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "height": "100",
          "src": undefined,
          "width": "100",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
